partner_communication/models/communication_job.py

```diff
@@ -390,8 +396,14 @@ def refresh_text(self, refresh_uid=False):
                 fields = self.env['mail.compose.message'].with_context(
                     lang=lang).get_generated_fields(
                     job.email_template_id, [job.id])
+                body = job.report_id.render_qweb_html([job.id])
+                body_html = tools.html_sanitize(body[0],
+                                                sanitize_attributes=True,
+                                                sanitize_style=True,
+                                                strip_classes=True)
                 job.write({
-                    'body_html': fields['body_html'],
+                    'body_html': fields['body_html'] if len(fields['body_html']) > 50
```

Comment:
I don't understand what you are trying to do here.

Reply:
It was sh*t, I changed it